aws-federation/env/dev_roleName

```diff
@@ -0,0 +1 @@
+SungardAS-aws-services-feder-LambdaFunctionIAMRole-1GY1VIJ5NKQ5C
```

Comment:
use a key-value pair. e.g. role='rolename'

Reply:
I cannot use it , as not able to parse json in Makefile

Reply:
use this ROLE := $(shell awk -F'[/=]' '{print $$2}' env/${ENV}) to read value from env file

Reply:
in env file use key-value pair like this
role=SungardAS-aws-services-feder-LambdaFunctionIAMRole-1GY1VIJ5NKQ5C

Reply:
ohk , yes I can use awk . doing now.

Reply:
populate staging and prod files too

Reply:
Makefile snippet

ifdef ENV
else

        ENV=dev;

endif

ifdef ROLE
else
        ROLE := $(shell awk -F'[/=]' '{print $$2}' env/${ENV})

endif

Reply:
Update readme